source/fundamentals/encrypt-fields.txt

```diff
@@ -45,10 +45,10 @@ uniquely.
 
 The MongoDB manual contains detailed information on the following {+qe+} topics:
 
-- To get started, see the :v6.0:`{+qe+} Quick Start </core/queryable-encryption/quick-start>`.
-- To learn how to use {+qe+}, see the :v6.0:`{+qe+} Fundamentals </core/queryable-encryption/fundamentals>`.
-- To learn how to integrate your implementation with a KMS, see the {+qe+} :v6.0:`{+qe+} Tutorials </core/queryable-encryption/tutorials>`.
-- To learn {+qe+} concepts, see the :v6.0:`{+qe+} Reference </core/queryable-encryption/reference>`.
+- To get started, see the :ref:`{+qe+} Quick Start <qe-quick-start>`.
+- To learn how to use {+qe+}, see the :ref:`{+qe+} Fundamentals <qe-fundamentals>`.
+- To learn how to integrate your implementation with a KMS, see the {+qe+} :ref:`{+qe+} Tutorials <qe-tutorials>`.
```

Comment:
This bullet point has some repetitive language, I'd take out the first "{+qe+}".
```suggestion
- To learn how to integrate your implementation with a KMS, see the :ref:`{+qe+} Tutorials <qe-tutorials>`.
```